apps/client/src/assets/i18n/fr-FR.json

```diff
@@ -1533,7 +1564,7 @@
     "Remove_from_basket": "Retirer du panier",
     "Search": "Trouver des recettes",
     "Generate_list": "Ajouter à une liste",
-    "Description": "Saisissez les objets que vous avez et cliquez sur « Trouver des recettes » pour obtenir une liste de recettes possibles pour fabriquer et générer une liste d'artisanat à partir des résultats.",
+    "Description": "Input the items you have and click \"Find recipes\" to get a list of possible recipes to craft and easily generate a crafting list from the results.",
```

Comment:
This was in french, why did you change it to english?

Reply:
This has been done by crowdin, because I fixed a typo in the original english translation and it reset the other languages to that as it can't guess if the change was minor or major for the meaning of the sentence.